conf/nomad/redash_worker.hcl

```diff
@@ -20,7 +20,7 @@ job "redash-worker" {
               local_bind_port = 6379
             }
             upstreams {
-              destination_name = "redash-postgres-service"
+              destination_name = "${postgres_service_name}"
               local_bind_port = 5432
```

Comment:
(not necessary) You might use output variables instead
```suggestion
              local_bind_port = "${postgres_port}"
```

Reply:
Yes! I will implement that.